docs/policies/security_incident_management_policy.md

```diff
@@ -0,0 +1,243 @@
+|Policy and strategy| A/ R| C| I|I|I|I|I|I|I|
+|Awareness| I|I| R|I|I|R|I|C / I||
+ 
+ *P1 Escalation Teams include the National Cyber Security Centre (NCSC), the Gold Team, BAE Systems and Government Security Group (GSG) who may be consulted / informed during a P1 incident (see Appendix 1, Criticality Matrix) if necessary.
+
+## Exceptions 
+
+There are no acceptable exceptions for this policy.
+Service Owners (SO) are responsible for ensuring that services are designed and operated in line with this policy. They are also responsible for ensuring that all users of their services understand their obligation to report information security incidents and have a mechanism to do so.
+
+## Security incident plan 
+
+The Cyber and Information Security division must develop and maintain the framework that all incident response plans must adhere to. All DfE incident response plans must:
+
+* be documented and approved
+* include an incident response process
+* be reviewed annually or following significant changes within DfE, if necessary 
+* a review may also occur following an incident or planned training exercise
+
+They must also include: 
+
+* a named incident manager and suitable back up incident managers assigned during working hours
+* if an external party is the incident manager, then an internal individual must be specified to oversee the response process
+* contact information 
+ - contact information for parties that need to be informed of security incidents must be maintained by the SIM team - internal contact information may be collected from DfE data sources, where appropriate 
+* address any regulatory or other compliance requirements
+* address communication options (for example, calls and emails) during an incident 
+
+## Security incident process
+
+The SIM team must develop and maintain a written process to enable the reporting of security incidents. This must include:
+
+* primary and secondary methods for reporting (including out of hours)
+* the minimum information needed
+* timeframes for reporting incidents
+
+At a minimum, the incident reporting process must be reviewed on an annual basis or following significant changes within DfE. 
+
+## Security incident reporting
+
+The SIM team must ensure that any incident reported which could potentially be a cyber incident or affect the security of information is managed appropriately, whether that is by the SIM team themselves or by portfolios and services ‘in house.’
+The incident must be reported through a formal security incident management process.
+The incident must be reported immediately, wherever practical, even if unconfirmed or when full details are not known, but always within 24 hours of discovery. If an incident is not reported within 24 hours the reporter must provide an explanation for the delay.
+The process must be simple, clear, and easy to follow. It should follow the below guidelines:
+
+•	use a single point of contact for reporting of incidents
```

Comment:
This has a copy paste error, needs to be `*` or `-` rather than `•`

Reply:
Sorry, thought I'd fixed all these!